test/src/serviceEndToEnd.test.ts

```diff
@@ -238,17 +251,18 @@ describe("Service end-to-end", () => {
         }
     }).timeout(3000);
 
-    
     it("create channel, relay twice throws error trigger dispute", async () => {
         const data = KitsuneTools.encodeTriggerDisputeData();
         const appRequest = appointmentRequest(data, account0, oneWayChannelContract.address, 0);
 
         const res = await request.post(`http://${nextConfig.hostName}:${nextConfig.hostPort}/appointment`, {
             json: appRequest
         });
-        expect(request.post(`http://${nextConfig.hostName}:${nextConfig.hostPort}/appointment`, {
-            json: appRequest
-        })).to.eventually.be.rejected;
+        expect(
+            request.post(`http://${nextConfig.hostName}:${nextConfig.hostPort}/appointment`, {
+                json: appRequest
+            })
+        ).to.eventually.be.rejected;
```

Comment:
I don't think this works as expected, as I had this problem before; this will likely pass regardless of the outcome, as the test ends before the promise returns. To use the `to.eventually.` syntax, it should be the last condition of the test and be returned, e.g.:
```
    return expect(something).to.eventually.be.rejected;
```
See https://www.chaijs.com/plugins/chai-as-promised/ for more on that.

I just checked that there are multiple other places in the codebase using this incorrectly, opening a separate ticket.

Reply:
I'll fix this as part of #265